### ISSUE TITLE
Logo and Banner

### ISSUE THREAD

**Issue by RossComputerGuy** [RossComputerGuy/stardustos#4]
Anyone who wants to put their idea for a concept logo/banner or an actual logo/banner can reply to this thread here. Also discuss about what you think the logo/banner should look like.

**Comment by Enstrayed** [comment]
I have a couple of designs **these are drafts, not final designs**
![image](https://user-images.githubusercontent.com/48845980/68270114-2dbc9880-0011-11ea-9042-9de4b7c0a8d8.png)
![image](https://user-images.githubusercontent.com/48845980/68270134-3a40f100-0011-11ea-8b93-e8018d8ecdc2.png)

**Comment by RossComputerGuy** [comment]
Top left looks good, I think it should mention NewLand because I like the idea that the kernel is mentioned.

**Comment by Enstrayed** [comment]
I agree, Newland is a fundamental part of Stardust and is unique to it. I think that it should be included in the logo.

**Comment by Enstrayed** [comment]
New Concept, including seperate banner **this is a draft**
![image](https://user-images.githubusercontent.com/48845980/68270686-fd75f980-0012-11ea-8e3c-e69e1d5b736d.png)

**Comment by RossComputerGuy** [comment]
Maybe use dots for stars and not the star shape. The banner could use more stars and have them be in the background and not the foreground.

**Comment by Enstrayed** [comment]
I took your suggested and made this draft. I put the dots as stars but I think I added too many, I might put less in a future draft. I used the names of the two projects as the base for the logo and banner. Stardust representing the stars in a night sky (add a moon perhaps?) and NewLand representing new ventures. **This is a draft, not a final revision.**
![image](https://user-images.githubusercontent.com/48845980/68326259-e9b4ac80-007f-11ea-9c1a-7f050cef485c.png)

**Comment by RossComputerGuy** [comment]
I was thinking like how the old Windows NT commercials looked, they just had a starry sky.

**Comment by Enstrayed** [comment]
Something like the startup graphic? 
![image](https://user-images.githubusercontent.com/48845980/68332021-e115a380-008a-11ea-8aa7-737597d994f1.png)
If so, I will see what I can do. Nice choice.

**Comment by RossComputerGuy** [comment]
Kinda, except I was thinking of having just the stars and the text saying "StardustOS (new line) With NewLand Kernel"

**Comment by RossComputerGuy** [comment]
I think using the tone of the colors for the badges in the README would be nice since I want to base the theme from that tone.
